test/mlir/test/gc/Transforms/Pipeline/gpu-pipeline.mlir

```diff
@@ -0,0 +1,18 @@
+// RUN: gc-opt %s --gc-gpu-pipeline -split-input-file | FileCheck %s
```

Comment:
should be located in the `gpu` subfolder (`test/mlir/test/gc/Transforms/GPU/...`)